packages/cashier/src/pages/withdrawal/withdrawal.tsx

```diff
@@ -74,6 +74,7 @@ const Withdrawal = observer(({ setSideNotes }: TWithdrawalProps) => {
         error: { setErrorMessage },
         willMountWithdraw,
     } = withdraw;
+    const { data, last_transaction } = useCryptoTransactions();
```

Comment:
Lift up the `useCryptoTransactions` to use the same data between `RecentTransaction` and `CryptoTransactionsHistory`.